### PR TITLE
Issue catalog clean up 2

### PR DESCRIPTION
**Pull request by fritzschoff**

### CODE REVIEW

File: libs/movie/src/lib/movie/marketplace/shell/shell.component.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import { Component, OnInit, ChangeDetectionStrategy, Input } from '@angular/core';
+import { Component, ChangeDetectionStrategy, Input, ViewChild, ElementRef } from '@angular/core';
 import { Router, RouterOutlet } from '@angular/router';
-import { Observable } from 'rxjs';
+import { Observable, } from 'rxjs';
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/marketplace/shell/shell.component.ts:3]: remove ","

File: package.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
     "deploy:functions:config": "ts-node -P tools/tsconfig.tools.json tools/scripts/deploy-function-config.ts",
     "firebase:serve": "concurrently --kill-others \"npm run build:functions --watch\" \"firebase serve --only functions\"",
     "firebase:shell": "concurrently --kill-others \"npm run build:functions --watch\" \"firebase functions:shell\" --raw",
-    "firebase:emulator": "firebase functions:config:get > .runtimeconfig.json && firebase emulators:start --only firestore,functions,auth,pubsub --import ./.firebase/firestore",
+    "firebase:emulator": "firebase functions:config:get > .runtimeconfig.json && firebase emulators:start --only firestore,functions,auth,pubsub --import ./.firebase/firestore --export-on-exit",
```
[the remark this answers is not in the document]
Comment (phpgeekfr) [reply package.json:32]: Yes, my bad ;)

File: apps/catalog/catalog/src/app/guards/deals.guard.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { CanActivate } from '@angular/router';
+import { Injectable } from '@angular/core';
+import { OrganizationQuery } from '@blockframes/organization/+state';
+
+@Injectable({ providedIn: 'root' })
+export class CatalogDealsGuard implements CanActivate {
```
[the remark this answers is not in the document]
Comment (pldespaigne) [reply apps/catalog/catalog/src/app/guards/deals.guard.ts:6]: I don't remember about that ^^
but there is indeed an app guard at `libs\utils\src\lib\routes\app.guard.ts`

[the remark this answers is not in the document]
Comment (fritzschoff) [reply apps/catalog/catalog/src/app/guards/deals.guard.ts:6]: I created the app guard :D . but currently this logic is app specific cause we got this page only accessible by a buyer. 
If you want to have it in a lib I can give it a name like OnlyBuyerGuard ?

File: libs/movie/src/lib/movie/marketplace/shell/shell.component.ts [hunk 4]
```diff
@@ -14,21 +14,30 @@ import { RouteDescription } from '@blockframes/utils/common-interfaces/navigatio
...
-    ) {}
+  ) { }
 
   ngOnInit() {
     this.movie$ = this.movieQuery.selectActive();
   }
 
+  scrollIntoView() {
+    /* Don't want to trigger the animation when the user just arrived on the page */
+    if (this.navClicked) {
```
[the remark this answers is not in the document]
Comment (fritzschoff) [reply libs/movie/src/lib/movie/marketplace/shell/shell.component.ts:36]: cause I listen on (@routerAnimation.start)

File: apps/catalog/catalog/src/app/dashboard/title/list/list.component.ts [hunk 5]
```diff
@@ -52,8 +53,9 @@ export class TitleListComponent implements OnInit {
   initialColumns = ['title', 'sales', 'receipt', 'status'];
   titles$: Observable<TitleView[]>;
   filter = new FormControl();
-  filter$: Observable<StoreType> = this.filter.valueChanges.pipe(startWith(this.filter.value));
-  movies$: Observable<Movie[]>;
+  filter$: Observable<StoreStatus> = this.filter.valueChanges.pipe(startWith(this.filter.value));
```
[the remark this answers is not in the document]
Comment (fritzschoff) [reply apps/catalog/catalog/src/app/dashboard/title/list/list.component.ts:56]: sorry, don't get this sentence

**Review by GrandSchtroumpf — APPROVED** [review]

File: apps/catalog/catalog/src/app/dashboard/home/home.component.html [hunk 6]
```diff
@@ -1,50 +1,48 @@
...
-          <a routerLink="../import" mat-stroked-button color="primary">Import your titles in bulk</a>
-        </mat-card>
-      </article>
-      <h3>Or</h3>
-      <button (click)="openIntercom()" mat-flat-button color="primary">
-        Contact Archipel Team
-      </button>
-    </section>
-  </ng-template>
+<ng-container *ngIf="(movies$ | async).length else noMovie">
```
[the remark this answers is not in the document]
Comment (fritzschoff) [reply apps/catalog/catalog/src/app/dashboard/home/home.component.html:1]: cause it loads for ever and will never finish. Analytics

**Pull request merged**